admin_manual/configuration_files/primary_storage.rst

```diff
@@ -182,3 +182,39 @@ all files for that user in their corresponding bucket.
 
 You can find out more information about upscaling with object storage and Nextcloud in the
 `Nextcloud customer portal <https://portal.nextcloud.com/article/object-store-as-primary-storage-16.html>`_.
+
+
+------------------------
+SSE-C encryption support
+------------------------
+
+Nextcloud added support for server side encryption (also known as SSE-C) with AWS.
+
+This means Nextcloud can encrypt files such storage, using a client provided key, the moment they are uploaded. The decryption key is provided by the client. We are using the `SSE-C provided by AWS <http://docs.aws.amazon.com/AmazonS3/latest/dev/ServerSideEncryptionCustomerKeys.html>`_ and to do this for Amazon AWS, the client has to specify exact information, so these options we provide in Nextcloud config:
```

Comment:
I am not sure about that one.

- The files are encrypted on the bucket side, but it is not clear in `the moment they are uploaded.`
- Not sure about `a client provided key` and ` decryption key is provided by the client`. It seems that we are talking about two keys. But there is only one. Right ?
- This paragraph might go too much in the implementation details, and i don't think that an admin care about that

```suggestion
The encryption and decryption appens on the S3 bucket side with a key provided by the Nextcloud server.
```